From f485ec4f9fc92f629c279190e60998417c0653a4 Mon Sep 17 00:00:00 2001
From: Hien Dang <hien.dang.eb@rvc.renesas.com>
Date: Thu, 26 May 2016 17:08:10 +0700
Subject: [PATCH 154/909] soc: renesas: rcar_ems_ctrl: Fix coding error when
 initializing static var to false

commit aa3f9197a7c57bba1ad81a3a5495e62b8f2e7f63 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Without this patch checkpatch script will throw out below error.
        ERROR: do not initialise statics to false
This patch will fix it.

Signed-off-by: Hien Dang <hien.dang.eb@rvc.renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar_ems_ctrl.c | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/drivers/soc/renesas/rcar_ems_ctrl.c b/drivers/soc/renesas/rcar_ems_ctrl.c
index 07c6250b096c..5d10f3afea77 100644
--- a/drivers/soc/renesas/rcar_ems_ctrl.c
+++ b/drivers/soc/renesas/rcar_ems_ctrl.c
@@ -42,12 +42,6 @@ static int ems_poll;
 static int thermal_zone_num;
 static struct thermal_zone_device *thermal_zone[EMS_THERMAL_ZONE_MAX];
 
-#ifdef CONFIG_RCAR_THERMAL_EMS_ENABLED
-static bool ems_enabled = true;
-#else
-static bool ems_enabled = false;
-#endif
-
 static int rcar_ems_notify(unsigned long state, void *p)
 {
 	return raw_notifier_call_chain(&rcar_ems_chain, state, p);
@@ -117,7 +111,7 @@ static int __init rcar_ems_ctrl_init(void)
 	struct thermal_zone_device *zone;
 	u32 value;
 
-	if (ems_enabled == false) {
+	if (!IS_ENABLED(CONFIG_RCAR_THERMAL_EMS_ENABLED)) {
 		pr_err("thermal emergency: disabled\n");
 		return 0;
 	}
@@ -258,7 +252,7 @@ static int __init rcar_ems_cpu_shutdown_init(void)
 	int cpu;
 	struct device_node *cpu_node;
 
-	if (ems_enabled == false)
+	if (!IS_ENABLED(CONFIG_RCAR_THERMAL_EMS_ENABLED))
 		return 0;
 
 	cpumask_clear(&target_cpus);
-- 
2.17.1

